rewrite/src/main/java/vct/col/rewrite/SimplifyNestedQuantifiers.scala

```diff
@@ -518,6 +518,8 @@ case class SimplifyNestedQuantifiers[Pre <: Generation]() extends Rewriter[Pre]
 
   case class Pointer[G](index: Expr[G], subnodes: Seq[Node[G]], array: Expr[G]) extends Subscript[G]
 
+  case class Sequence[G](index: Expr[G], subnodes: Seq[Node[G]], array: Expr[G]) extends Subscript[G]
+
```

Comment:
@niomaster This is the only change of interest, and it's not actually related to the intention of the silicon-progress-tracing branch. I think it's also in #950. So as far as I'm concerned this branch can be deleted.